fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -1198,6 +1198,21 @@ public void testAgentGroupWithHashUris() throws Exception {
         assertEquals(HttpStatus.SC_FORBIDDEN, getStatus(getUnauthorized));
     }
 
+    @Test
+    public void testAgentGroupWithMembersAsURIs() throws Exception {
+        System.setProperty(USER_AGENT_BASE_URI_PROPERTY, "http://example.com/");
+        ingestTurtleResource("fedoraAdmin", "/acls/agent-group-list-with-member-uris.ttl",
+                             serverAddress + "/rest/agent-group-list-with-member-uris");
+        final String authorized = ingestObj("/rest/agent-group-with-vcard-member-as-uri");
+        ingestAcl("fedoraAdmin", "/acls/agent-group-with-vcard-member-as-uri.ttl", authorized + "/fcr:acl");
+        //check that test user is authorized to write
+        final HttpPut childPut = new HttpPut(authorized + "/child");
+        setAuth(childPut, "testuser");
+        assertEquals(HttpStatus.SC_CREATED, getStatus(childPut));
+
+        System.clearProperty(USER_AGENT_BASE_URI_PROPERTY);
```

Comment:
Although the resetting of System Properties is handled by:
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java#L76

Being explicit about it here is not too harmful.

Reply:
Ah - I did't realize @Rule was taking care of that.  I'll remove it.